assets/css/edd-admin.css

```diff
@@ -2431,6 +2418,55 @@ body.download_page_edd-reports {
 	margin-bottom: 20px;
 }
 
+div.edd-reports-export {
+	display: grid;
+	display: -ms-grid;
+	grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+	grid-gap: 20px;
+}
+
+.edd-reports-wrapper .postbox {
+	margin-bottom: 0;
+}
+
+.edd-export-sales-earnings {
+	grid-template-columns: 1fr;
+}
+
+.edd-from-to-wrapper {
+	margin-right: 0;
+}
+
+input.edd-export-start,
```

Comment:
Setting the widths on these inputs works, but I'm wondering about removing this entire rule and using flexbox on the container instead, something like:

```css
.edd-from-to-wrapper {
    margin-right: 0;
    display: flex;
    width: 100%;
    margin-bottom: 16px;
}

#edd-start-wrap,
#edd-end-wrap {
    flex-grow: 1;
}
```

Really it would be better to set a class on the `edd-start-wrap` and `edd-end-wrap` elements and use that. Also pretty sure the `edd-from-to-wrapper` would be better as a `div` instead of a `span` (in the markup), at least with this new styling--maybe it needed to be a span when everything was on one line.

Reply:
Looking at the rest of the codebase, you could change these to `div`, I think, but the CSS would need to be more specific to manage just the exports, as this from-to input group is replicated elsewhere and styling just `.edd-from-to-wrapper` would do bad things to that!

Reply:
I couldn't get this to work with `flex-grow: 1`.  My solution feels kinda hacky, so please let me know if there is a better way to do this with flexbox.